python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -320,11 +332,15 @@ def setDefaults(self):
                                          "base_PsfFlux",
                                          "base_CircularApertureFlux",
                                          "base_ClassificationSizeExtendedness",
+                                         "base_CompensatedTophatFlux",
                                          ]
         self.star_measurement.slots.psfShape = "ext_shapeHSM_HsmPsfMoments"
         self.star_measurement.slots.shape = "ext_shapeHSM_HsmSourceMoments"
         # Only measure the apertures we need for star selection.
         self.star_measurement.plugins["base_CircularApertureFlux"].radii = [12.0]
+        self.star_measurement.plugins["base_CompensatedTophatFlux"].apertures = [12]
+
+        self.star_normalized_calibration_flux.do_measure_ap_corr = False
```

Comment:
Please add a comment as to why you set this (I think I know, but it wouldn't be obvious to someone just looking at setDefaults).